osu.Framework/Input/Events/KeyBindingPressEvent.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Input.States;
+
+namespace osu.Framework.Input.Events
+{
+    /// <summary>
+    /// An event representing a press of a key binding combination.
+    /// </summary>
+    /// <typeparam name="T">The action type.</typeparam>
+    public class KeyBindingPressEvent<T> : KeyBindingEvent<T> where T : struct
+    {
+        public readonly bool Repeat;
```

Comment:
Probably should add xmldoc here just to be in line with all the other documented events.

Reply:
```suggestion
        /// <summary>
        /// Whether this is a repeated event resulted from a held keyboard key.
        /// </summary>
        public readonly bool Repeat;
```
Does this read fine?

Reply:
Seems fine.